admin/config-ui/fields/class-field-multiple-authors.php

```diff
@@ -67,14 +67,15 @@ public function get_data() {
 	public function set_data( $data ) {
 		$value = ( $data === 'yes' );
 
-		$option                           = WPSEO_Options::get_option( 'wpseo' );
-		$option['has_multiple_authors'] = $value;
+		// Set multiple authors.
+		$result_multiple_authors = WPSEO_Options::save_option( 'wpseo', 'has_multiple_authors', $value );
 
-		update_option( 'wpseo', $option );
+		/*
+		 * Set disable author archives. Inverse the value for this, because the when multiple authors is true,
```

Comment:
Could you please improve this documentation? Currently, it doesn't make a lot of sense.